hal/sparcv8leon3/_traps.S

```diff
@@ -43,9 +43,9 @@
 
 #define RESET \
 	mov	%g0, %g4; \
-	sethi %hi(_init), %g4; \
-	jmp %g4 + %lo(_init); \
-	nop;
+	nop; \
+	call _init; \
```

Comment:
Why replace with call?

Side note - this is somewhat riscy strategy, same HW can be in an unexpected state after this kind of warm reset. Is there's no way to generate MCU reset in software?

Reply:
call is pc relative, so this allows for correct jump even when we're loaded to incorrect address

Reply:
It is indeed, you're right